src/scripts/fetchStatsFromGA.js

```diff
@@ -0,0 +1,437 @@
+// eslint-disable no-console
+// TODO: consider the edge case when the cron job runs at midnight
```

Comment:
Some explanation on how to execute the script here (or in `README.md` instead) would be great.

Newly introduced env vars in `.env.sample` needs some documentation in here (or in `README.md` instead).